tutorials/fundamentals/pop.md

```diff
@@ -0,0 +1,22 @@
+The `pop()` functions removes the last element from the array it is called upon and returns the popped element.
+
+```javascript
+const array = [1, 2, 3, 4, 5, 6];
+array.pop(); // 6;
+array; // 1, 2, 3, 4, 5
+```
+
+This function shortens the length of the array and returns `undefined` if the array is empty, similarly to [`shift()`](/tutorials/fundamentals/shift).
```

Comment:
"This function shortens the length of the array" -> "This function reduces the length of the array by 1" and move it to line 2.

"And returns `undefined`" -> "`pop()` returns `undefined`"

"similarly to" -> "like"